.circleci/run.sh

```diff
@@ -104,9 +105,7 @@ setup_repos() {
 coverage()
 {
     # load environment for bootstrap compiler
-    if [ -f ~/dlang/install.sh ] ; then
-        source "$(CURL_USER_AGENT=\"$CURL_USER_AGENT\" bash ~/dlang/install.sh dmd-$HOST_DMD_VER --activate)"
-    fi
+    source "$(CURL_USER_AGENT=\"$CURL_USER_AGENT\" bash ~/dlang/install.sh dmd-$HOST_DMD_VER --activate)"
```

Comment:
What'S up with all these changes?

Reply:
The distinction was made when we used our own D CircleCi docker image (with the host compiler pre-installed). We no longer do that, hence the conditional check.

Reply:
I see, thanks for the explaination!